ngraph/test/opset1.cpp

```diff
@@ -28,6 +28,8 @@
 using namespace std;
 using namespace ngraph;
 
+NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
Why are we deprecating tests for opset1?

Reply:
For Windows build.
In tests we call destructor for FusedOp and Windows compiler throws an warning.